src/java/picard/illumina/IlluminaBasecallsToSam.java

```diff
@@ -95,11 +95,36 @@
  * @author mccowan@broadinstitute.org
  */
 @CommandLineProgramProperties(
-        usage = IlluminaBasecallsToSam.USAGE,
-        usageShort = IlluminaBasecallsToSam.USAGE,
+        usage = IlluminaBasecallsToSam.USAGE_SUMMARY + IlluminaBasecallsToSam.USAGE_DETAILS,
+        usageShort = IlluminaBasecallsToSam.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class IlluminaBasecallsToSam extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Transforms raw Illumina sequencing data into an unmapped SAM or BAM file."  ;
+    static final String USAGE_DETAILS = "<p>The IlluminaBaseCallsToSam program collects, demultiplexes, and sorts reads across all of the " +
+            "tiles of a lane via barcode to produce an unmapped SAM/BAM file.  " +
+            "An unmapped BAM file is often referred to as a uBAM.  All barcode, sample, and library data is provided in the LIBRARY_PARAMS file.  " +
+            "Within each barcode-specific directory, subdirectories are created which contain all of the first reads, second reads, and index reads from a run.</p> " +
+            "" +
+            "<p>The BARCODES_DIR file is produced by the " +
```

Comment:
Isn't BARCODES_DIR a directory? Name and definition below both say so. Don't pay attention to the `public File [...]` part in the code -- that's just because java doesn't differentiate between files and paths in this context.